lib/themes.dart

```diff
@@ -0,0 +1,7 @@
+import 'package:flutter/material.dart';
+
+const String kAppName = "Grievance Portal";
```

Comment:
Why is app name `string` constant stored in `themes.dart` file. Use `constants.dart` for the same.